lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -727,7 +727,6 @@ def test_dates_tab_link_render(self, url_name, mfe_active):
             'class="dates-tab-link"',
             'View all course dates</a>',
         ]
-        # The url should change based on the mfe being active.
```

Comment:
Accidental deletion? Or is this not a useful comment?

Reply:
accidental/ my previous edit of this function didn't need it but will put this back in for the updated one!